app/components/admin/editions/audit_trail_entry_component.html.erb

```diff
@@ -1,8 +1,13 @@
-<li class="app-view-editions-audit-trail-entry__list-item govuk-!-margin-bottom-4">
+<div class="app-view-editions-audit-trail-entry__list-item govuk-!-margin-bottom-4">
+  <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0"><%= action %></p>
```

Comment:
The 'action' is wrapped in a `<p>` tag, but the commit 7fe2b9add13e17136672982b196c364e509b37b2 says it should be a `<h4>`.

Does this need changing? 🤔 

<img width="632" alt="Commit message" src="https://user-images.githubusercontent.com/7735945/212053815-60a3e90a-d904-478a-8585-bbe63c8930dc.png">

Reply:
oh sorry that just meant to be for a editorial remark. I'll update the commit message now